### ISSUE TITLE
Migrate security advisories

### ISSUE THREAD

**Issue by yanesca** [ARMmbed/mbedtls#3211]
## Background:

As a consequence of the transition to TF.org we would like to move to the new infrastructure and as part of this we are going to migrate the information available on the website. As a preparation for publishing security advisories at the new location, we migrate the security advisories first. Migrating any other content is out of scope for this task.

**Prerequisite:** #3643

## Task breakdown

- [x] Copy all existing security advisories from [the website](https://tls.mbed.org/tech-updates/security-advisories) to the documentation site (https://github.com/ARMmbed/mbedtls-docs)
~- [ ] When done, update the security advisory list pages with a link and an announcement to direct visitors to the new location. (On editing the website see the Release Process confluence page.)~

**Comment by yanesca** [comment]
@bensze01 Could you please post an update with a short summary of the current status of this task?

**Comment by bensze01** [comment]
Current status:
It was decided that the security advisories should probably move to the same platform as the rest of the documentation (tentatively Read the Docs). That system handles changes to the documentation through the regular SCM tools, so accidental modification shouldn't be an issue.

The results of the Read the Docs investigation are summarized in #3643 , which this issue will now depend on.

**Comment by yanesca** [comment]
An efficient way to do it would be to ask the devops team to retrieve the markdown sources from the database and submit them in a PR.

**Comment by yanesca** [comment]
For better parallelisation of work, it is best if we only update the website with the link when the documentation framework is up and running.

**Comment by davidhorstmann-arm** [comment]
I didn't notice that the second part of the task (update the website) was removed. Since this is no longer required, this issue can be closed.
